charts/kong/example-values/doc-examples/quickstart-enterprise-licensed-aio.yaml

```diff
@@ -113,11 +113,6 @@ env:
       secretKeyRef:
         key: pg_host
         name: kong-config-secret
-  pg_password:
-    valueFrom:
-      secretKeyRef:
-        key: password
-        name: kong-config-secret
```

Comment:
`KONG_PG_PASSWORD` will be loaded from secret generated by postgresql

Reply:
How will this work exactly? Do you mind leaving a comment here 🤔 ?